fedimint-client/src/db.rs

```diff
@@ -19,7 +19,7 @@ pub enum DbKeyPrefix {
     ChronologicalOperationLog = 0x2d,
     CommonApiVersionCache = 0x2e,
     ClientConfig = 0x2f,
-    ClientInviteCode = 0x30,
+    ClientInviteCode = 0x30, // Unused, we can get an invite code from the client config
```

Comment:
As long as this stays here future developers will know to clean out remnant data before re-using, but ideally we'd add a migration that removes all entries with that prefix, then we can also remove this enum variant.

Reply:
@joschisan what are your thoughts about this?

Reply:
I added a comment about cleaning out remnants before reusing and would not do it for now as it would need a new migration hook for non-module db migration and maybe we get a chance in the future to toss it out with less effort, should we ever make a breaking change to the client.